b2b-sample/guardio-insurance-sample-application/components/data/templates/enterprise-identity-provider.json

```diff
@@ -14,9 +14,82 @@
         "uri": ""
       }
     },
-    "description": "",
+    "description": "Enable login for users with their accounts in your existing identity providers via standard authentication protocols.",
     "federatedAuthenticators": {
-      "authenticators": [],
+      "authenticators": [
+        {
+          "authenticatorId": "RmFjZWJvb2tBdXRoZW50aWNhdG9y",
+          "isEnabled": true,
+          "properties": [
+            {
+              "key": "ClientId",
+              "displayName": "Client Id",
+              "description": "The client identifier value of the identity provider.",
+              "type": "STRING",
+              "displayOrder": 1,
+              "regex": ".*",
+              "isMandatory": true,
+              "isConfidential": false,
+              "options": [],
+              "defaultValue": "",
+              "subProperties": []
+            },
+            {
+              "key": "ClientSecret",
+              "displayName": "Client Secret",
+              "description": "The client secret value of the identity provider.",
+              "type": "STRING",
+              "displayOrder": 2,
+              "regex": ".*",
+              "isMandatory": true,
+              "isConfidential": true,
+              "options": [],
+              "defaultValue": "",
+              "subProperties": []
+            },
+            {
+              "key": "OAuth2AuthzEPUrl",
+              "displayName": "Authorization Endpoint URL",
+              "description": "The standard authorization endpoint URL obtained from the identity provider.",
+              "type": "STRING",
+              "displayOrder": 3,
+              "regex": ".*",
+              "isMandatory": true,
+              "isConfidential": false,
+              "options": [],
+              "defaultValue": "",
+              "subProperties": []
+            },
+            {
+              "key": "OAuth2TokenEPUrl",
+              "displayName": "Token Endpoint URL",
+              "description": "The standard token endpoint URL obtained from the identity provider.",
+              "type": "STRING",
+              "displayOrder": 4,
+              "regex": ".*",
+              "isMandatory": true,
+              "isConfidential": false,
+              "options": [],
+              "defaultValue": "",
+              "subProperties": []
+            },
+            {
+              "key": "callbackUrl",
+              "displayName": "Callback Url",
+              "description": "The WSO2 Identity Server URL to which the user needs to be redirected after completing the authentication at the identity provider. The identity provider needs to send the authorization code to this URL upon successful authentication.",
```

Comment:
Let's make sure a line does not exceed 120 characters.

Reply:
Ref: https://github.com/wso2/samples-is/pull/346#discussion_r951669697

Reply:
This is a template file, and IMO it is okay to exceed 120 characters in such files, as we have done the same in our other repos as well.

ex: 
- https://github.com/wso2-enterprise/asgardeo-apps/blob/58c7f91640fae55d4215c33a7075399aeabe7fe2/apps/console/src/extensions/identity-provider-templates/templates/saml/saml.json#L61